src/scss/base.scss

```diff
@@ -1,15 +1,16 @@
+
 @include oFontsInclude(BentonSans, normal);
 
-.o-ft-header {
+.o-header {
 	// 80 px for the primary container and 5 for an empty secondary or 25 for a secondary with content
 	min-height: 85px;
 	max-height: 125px;
 	font-family: oFontsGetFontFamilyWithFallbacks(BentonSans);
```

Comment:
Thanks Luke! I'll follow up the conversation with the design team because I believe they should own the vocabulary and the use cases.